rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/interpreter/internal_interpreter.kt

```diff
@@ -110,13 +111,13 @@ open class InternalInterpreter(
 
     fun logsEnabled() = logHandlers.isNotEmpty()
 
-    private val status = InterpreterStatus(globalSymbolTable, indicators)
+    private val dbFileMap = DBFileMap()
+
+    private val status = InterpreterStatus(globalSymbolTable, indicators, dbFileMap = dbFileMap)
```

Comment:
Now `dbFileMap `is in both `status `and `this`, I don't like having the same instance in two different places. If `dbFileMap `is necessary in `InterpreterStatus`, because in the `ExpressionEvaluation `we don't have `InterpreterCore`, then remove it from `InternalInterpreter`

Reply:
Done